### PR TITLE
Fixed mesh renderers of the masked enemy not being updated

### PR DESCRIPTION
**Pull request by 1A3Dev**
- Fixed mesh renderer list of the masked enemy not being updated after applying/clearing cosmetics.

### CODE REVIEW

**Comment by notnotnotswipez** [comment]
Does this fix anything?

**Comment by notnotnotswipez** [comment]
The masked enemies are instance based, not pooled. So theres really no reason to add the cosmetics to the meshrenderers unless theres some sort of other issue pertaining to that

**Comment by 1A3Dev** [comment]
> Does this fix anything?

Fixes compatibility with some other mods such as Masked Enemy Overhaul

**Comment by notnotnotswipez** [comment]
Is there a repository link or something similar of a mod that has a bug with this?

**Comment by 1A3Dev** [comment]
https://i.gyazo.com/55916d58b95d7a6e8c3cb8b1d08710bb.png
https://i.gyazo.com/01afce1d6d77b976f4831f7e77e411d4.png

TLDR it not being updated can cause there to be null references when executing EnableEnemyMesh

**Comment by notnotnotswipez** [comment]
I see, thank you for the fix. Will merge
